projects/ngx-d3timeline/src/lib/options/selectors/options.selectors.ts

```diff
@@ -20,20 +15,30 @@ export const selectResourceOrientation = createSelector(
   flipOrientation
 );
 
-export const selectAcivityPadding = createSelector(
-  selectTypeActivityPadding,
-  selectActivityOptionsPadding,
-  (typeActivityPadding, activityOptionsPadding) =>
-    getActivityPadding.bind(null, typeActivityPadding, activityOptionsPadding)
-);
+const selectActivityOption = <T>(
```

Comment:
you can do something like 

```const selectActivityOption = <TOption extends keyof ActivityOptions>(```

then its properly typed and you don't need to pass the type in when you call like on line 32

Reply:
done as part of #470